defaults.go

```diff
@@ -49,7 +49,6 @@ func empty(given interface{}) bool {
 	case reflect.Struct:
 		return false
 	}
-	return true
```

Comment:
This return is unreachable. Switch is handling all the cases (because of default) and each one of them are returning something then control will never reach outside switch.